notebooks/book2/03/linreg_post_pred_plot.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    def shp(x):
</pre><p><code>shp</code> is not meaningful name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>